ui/qt-app/ariadne-qt-app.cabal

```diff
@@ -0,0 +1,79 @@
+name:           ariadne-qt-app
+version:        0.2.0.0
+description:    Please see the README on Github at <https://github.com/serokell/ariadne#readme>
```

Comment:
Now that we have many packages, I think it'd be good to write something more concrete here. Not necessary in this PR, just speaking loudly.